server/sonar-server/src/main/java/org/sonar/server/issue/ws/IssueJsonWriter.java

```diff
@@ -65,12 +99,22 @@ public IssueJsonWriter(I18n i18n, Durations durations, UserSession userSession,
   }
 
   public void write(JsonWriter json, Issue issue, Map<String, User> usersByLogin, Map<String, ComponentDto> componentsByUuid,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "write" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)